packages/big-design/src/components/Worksheet/styled.tsx

```diff
@@ -0,0 +1,22 @@
+import { theme as defaultTheme } from '@bigcommerce/big-design-theme';
+import styled from 'styled-components';
+
+export const Table = styled.table`
+  border-collapse: collapse;
+  border-spacing: 0;
+  table-layout: fixed;
+  width: 100%;
+`;
+
+export const Header = styled.th`
+  background-color: ${({ theme }) => theme.colors.secondary10};
+  border: ${({ theme }) => `${theme.helpers.remCalc(0.5)} solid ${theme.colors.secondary30}`};
```

Comment:
🍹  I wonder if we need a "tableBorder" internal theme prop or something to provide this styling, it'll be easier to enforce, maintain and use